php/class-customize-snapshot-manager.php

```diff
@@ -225,7 +226,7 @@ public function add_snapshot_uuid_to_return_url() {
 			&&
 			$this->current_snapshot_uuid
 			&&
-			$this->is_theme_active()
+			$this->customize_manager->is_theme_active()
 			&&
```

Comment:
It's because the `setup_theme` action wasn't triggered in the test case, and so `\WP_Customize_Manager::$original_stylesheet` didn't get set. In 4.7 this variable gets set in `\WP_Customize_Manager::__construct()` whereas in 4.6 it got set in `\WP_Customize_Manager::setup_theme()`.